kubernetes-client/src/main/java/io/fabric8/kubernetes/client/KubernetesClientException.java

```diff
@@ -49,12 +49,16 @@ public int getCode() {
   }
 
   public static RuntimeException launderThrowable(Throwable cause) {
+    return launderThrowable("An error has occurred.", cause);
```

Comment:
Don't understand why this is needed Vs `new KubernetesClientException (msg, cause)`?